src/function/builtin.rs

```diff
@@ -148,6 +148,25 @@ pub fn builtin_function(identifier: &str) -> Option<Function> {
             let result_index = if arguments[0].as_boolean()? { 1 } else { 2 };
             Ok(arguments.swap_remove(result_index))
         })),
+        "some" => Some(Function::new(move |argument| {
+            let arguments = argument.as_tuple()?;
+            if let (Value::Tuple(a), b) = (&arguments[0].clone(), &arguments[1].clone()) {
```

Comment:
Using the bracket access operator may panic. I would first check if the number of arguments is exactly two. You can use `argument.as_fixed_len_tuple(2)?` in the previous line to do that automatically.

Reply:
Thanks! Applied. If no further detail, please let me know and I can resolve.